src/components/Site/clientSideRedirections.js

```diff
@@ -12,8 +12,9 @@ export default function (location) {
 
   if (location.pathname === '/webpack-options/') {
```

Comment:
Maybe `/webpack/options/` so in future we will add `/webpack-dev-server/options/`

Reply:
How about `https://webpack.js.org/webpack/?option=output/filename` `https://webpack.js.org/webpack-dev-server/?option=xxx`? It's more terse.

Or `https://webpack.js.org/redirect/webpack/?option=output/filename` which is more future proof in case we will be using `/webpack/` path on this site in future.